### PR TITLE
chore(bootstrap): refactor code

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**

refactoring
**Did you add tests for your changes?**
Nope
**If relevant, did you update the documentation?**
N/A
**Summary**
refactor code

**Does this PR introduce a breaking change?**

N/A
**Other information**

### CODE REVIEW

**Comment by jamesgeorge007** [comment]
cc @evenstensberg

File: lib/bootstrap.js [hunk 1]
```diff
@@ -14,22 +14,20 @@ const isCommandUsed = commands =>
 
 async function runCLI(cli, commandIsUsed) {
     let args;
-    const helpFlagExists = isFlagPresent(process.argv, 'help');
```
Comment (ematipico) [inline lib/bootstrap.js]: I would revert this change. As it is at the moment it's better because it's possible to test the function and it's a better functional approach where the function stays pure.

**Comment by jamesgeorge007** [comment]
\cc @evenstensberg

**Comment by jamesgeorge007** [comment]
\cc @evenstensberg

File: lib/bootstrap.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const cmdArgs = require('command-line-args');
 require('./utils/process-log');
 
 const isFlagPresent = (args, flag) => args.find(arg => [flag, `--${flag}`].includes(arg));
-const normalizeFlags = (args, command) => args.slice(2).filter(arg => arg.indexOf('--') < 0 && arg !== command.name && arg !== command.alias);
+const normalizeFlags = cmd => process.argv.slice(2).filter(arg => arg.indexOf('--') < 0 && arg !== cmd.name && arg !== cmd.alias);
```
Comment (ematipico) [inline lib/bootstrap.js]: Of course, also this should be reverted

Comment (jamesgeorge007) [reply lib/bootstrap.js]: Oops! yeah :sweat_smile:

File: lib/bootstrap.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ const cmdArgs = require('command-line-args');
 require('./utils/process-log');
 
 const isFlagPresent = (args, flag) => args.find(arg => [flag, `--${flag}`].includes(arg));
-const normalizeFlags = (args, command) => args.slice(2).filter(arg => arg.indexOf('--') < 0 && arg !== command.name && arg !== command.alias);
+const normalizeFlags = (args, cmd) => args.slice(2).filter(arg => arg.indexOf('--') < 0 && arg !== cmd.name && arg !== cmd.alias);
```
Comment (rishabh3112) [inline lib/bootstrap.js]: IMO changing `command` to `cmd` doesn't bring much of a value to the meaningfulness of the variable nor to the readability of code.

Comment (jamesgeorge007) [reply lib/bootstrap.js]: Just to stay consistent with rest of the code-base.
https://github.com/webpack/webpack-cli/blob/8a54b58e68d976c4265c0d466854d69f779a6650/lib/bootstrap.js#L11

Comment (jamesgeorge007) [reply lib/bootstrap.js]: https://github.com/webpack/webpack-cli/blob/8a54b58e68d976c4265c0d466854d69f779a6650/lib/bootstrap.js#L12
